### PR TITLE
{tools}[intel/2015a] vsc-mympirun-scoop 3.5.0 on top of SCOOP 0.7.1.1 (WIP)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/679

### CODE REVIEW

**Comment by boegel** [comment]
@JensTimmerman: please review?

**Comment by boegel** [comment]
Although the installation works, it seems that `myscoop` is broken with these versions of `vsc-base` and `vsc-mympirun`:

```
$ myscoop --sched=local --scoop-module=squares 10000 -ld
Usage: myscoop [options]

myscoop: error: no such option: --scoop-module
```

To be looked into, it's likely that `vsc-mympirun-scoop` will need an update first...

File: easybuild/easyconfigs/v/VSC-tools/VSC-tools-2.1.5-intel-2015a-scoop-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name = 'VSC-tools'
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/v/VSC-tools/VSC-tools-2.1.5-intel-2015a-scoop-Python-2.7.10.eb]: we should get this package to  change their name to all lowercase :p

**Comment by JensTimmerman** [comment]
looks ok, 
what's the output of `myscoop --help` and ` myscoop --debug --sched=local --scoop-module=squares 10000 ` (put debug first to debug the rest of the option parsing.) Also note that the `-l` option does not exist.

**Comment by boegel** [comment]
on hold until https://github.com/hpcugent/vsc-mympirun-scoop/pull/8 gets merged in

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/v/vsc-mympirun-scoop/vsc-mympirun-scoop-3.5.0beta-intel-2015b-Python-2.7.10.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+version = '3.5.0beta'
+
+homepage = 'http://hpcugent.github.com/vsc-mympirun/'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+# we build this to work with every python version
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = ['scoop_0.7.tar.gz']
+source_urls = ['https://github.com/boegel/vsc-mympirun-scoop/archive/']
```
Comment (boegel) [inline easybuild/easyconfigs/v/vsc-mympirun-scoop/vsc-mympirun-scoop-3.5.0beta-intel-2015b-Python-2.7.10.eb]: WIP, https://github.com/hpcugent/vsc-mympirun-scoop/pull/8 should get merged first, and these have to fixed accordingly

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0805be9b0433f2ef8b76 for a full test report.

**Comment by migueldiascosta** [comment]
@boegel ping?

**Comment by boegel** [comment]
blocked by https://github.com/hpcugent/vsc-mympirun-scoop/pull/8, which is closed, so I'm closing this too...

**Pull request closed without merging**